realm/realm-library/src/main/java/io/realm/internal/TableOrView.java

```diff
@@ -328,8 +328,6 @@
 
     TableView findAllDouble(long columnIndex, double value);
 
-    TableView findAllDate(long columnIndex, Date value);
```

Comment:
Is this because of it not being supported by Core yet or is there some other reason? Perhaps just add a FIXME if it is the first?